Neos.SiteKickstarter/Classes/Generator/FluidTemplateGenerator.php

```diff
@@ -1,26 +1,21 @@
 <?php
-namespace Neos\SiteKickstarter\Service;
 
-/*
- * This file is part of the Neos.Kickstarterer package.
- *
- * (c) Contributors of the Neos Project - www.neos.io
- *
- * This package is Open Source Software. For the full copyright and license
- * information, please view the LICENSE file which was distributed with this
- * source code.
- */
+namespace Neos\SiteKickstarter\Generator;
 
 use Neos\Flow\Annotations as Flow;
 use Neos\Flow\Package\PackageManager;
+use Neos\SiteKickstarter\Service\FusionRecursiveDirectoryRenderer;
 use Neos\Utility\Files;
 use Neos\ContentRepository\Domain\Repository\ContentDimensionRepository;
 use Neos\ContentRepository\Utility;
+use Neos\SiteKickstarter\Annotation as SiteKickstarter;
 
 /**
  * Service to generate site packages
+ *
+ * @SiteKickstarter\SitePackageGenerator("Fluid Basic")
  */
-class GeneratorService extends \Neos\Kickstarter\Service\GeneratorService
+class FluidTemplateGenerator extends AbstractSitePackageGenerator
```

Comment:
Should we really offer a Fluid template generator? I guess when asked, we recommend not to use fluid for site templating anymore.